desktop/apps/auction2/client/reducers.js

```diff
@@ -30,6 +30,7 @@ const initialState = {
   followedArtistRailPage: 1,
   followedArtistRailSize: 4,
   initialMediumMap: [],
+  isOpen: sd.AUCTION && sd.AUCTION.is_open,
```

Comment:
I've noticed we're now having to guard against `sd.AUCTION` in a few places -- since we're in the `/auction2` namespace shouldn't it always be there? Or does this relate to the subtle differences between `sd.SALE` and `sd.AUCTION`?